### PR TITLE
Added a command before submission to change to the same directory tha…

### PR DESCRIPTION
**Pull request by adamfitzgibbon**
…t the job file is located in.

Addresses #202

### CODE REVIEW

File: HCCGo/app/js/ConnectionService.js [hunk 1]
```diff
@@ -236,11 +236,13 @@ connectionModule.factory('connectionService',['$log', '$q', '$routeParams', '$lo
    var submitJob = function(location) {
       var deferred = $q.defer();
       $log.debug("Running command: " + 'sbatch ' + location);
-      runCommand('sbatch ' + location).then(function(data) {
-        deferred.resolve(data);
-      }, function(data) { // thrown on failure
-        $log.log("Error log: " + data)
-        return deferred.reject("An error occurred when submitting the job.");
+      runCommand('cd ' + path.dirname(location)).then(function() {
+        runCommand('sbatch ' + location).then(function(data) {
```
Comment (djw8605) [inline HCCGo/app/js/ConnectionService.js]: Have you tested this?  I thought each runCommand was independent of all the other runCommands.  So the affect of `cd` won't be seen in the second `runCommand`

Comment (adamfitzgibbon) [inline HCCGo/app/js/ConnectionService.js]: I was thinking about this but I wasn't sure how to test it. Can two commands be run in the same runCommand?

**Comment by djw8605** [comment]
You can do 2 commands separated with a semi colon. 

-Derek

**Comment by adamfitzgibbon** [comment]
Okay that should fix it then.

**Pull request merged**